avm/err.txt

```diff
@@ -0,0 +1,266 @@
+warning: unused import: `futures::future::join_all`
```

Comment:
I'm guessing this one sneaked in :)

Reply:
heh, i forgot to get rid of the cargo warnings and the temp files. fixed!